scripts/scripts.js

```diff
@@ -76,7 +97,10 @@ async function loadPage() {
   const config = setConfig({ ...CONFIG, miloLibs });
   await loadArea();
   await headerModal();
-  setTimeout(() => { loadMartech(); }, 3000);
+  setTimeout(() => {
+    loadMartech();
+    loadProfile();
```

Comment:
shouldn't we add this before martech? does it need to be in delayed or can it be above the setTimeout (delayed)?